mods/voidmulti.sc2mod/base.sc2data/gamedata/abildata.xml

```diff
@@ -133,7 +141,7 @@
                 <DurationArray index="Duration" value="0.5"/>
             </SectionArray>
             <SectionArray index="Stats">
-                <DurationArray index="Duration" value="0.2221"/>
+                <DurationArray index="Duration" value="0"/>
```

Comment:
**undocumented**: Hydralisk's Unburrow stats transition duration is now changed to 0 from 0.2221
Does this cause unburrowing units to become "visible" immediately instead of after ~150ms?